kate/proof/src/lib.rs

```diff
@@ -83,7 +82,6 @@ pub fn kc_verify_proof(
 
 	Ok(ProofVerification {
 		status,
-		public_params_hash: hex::encode(sp_core::blake2_128(&raw_pp)),
-		public_params_len: hex::encode(raw_pp).len(),
+		public_params: raw_pp,
```

Comment:
Should we keep a log of hash+len of public params in debug log level? Might be useful. We need not use `blake2`. Any standard hash function like SHA256 or even MD5 might suffice.